internal/db/migrations/005_reservations.sql

```diff
@@ -25,7 +25,7 @@ CREATE TABLE aws_reservation_details
   source_id INTEGER NOT NULL,
   instance_type TEXT NOT NULL,
   amount INTEGER NOT NULL,
-  ami TEXT NOT NULL,
+  image_id TEXT NOT NULL,
```

Comment:
I think that here we want to store AMI, we want to exchange image_id before we create aws_reservation_details IMHO :thinking:

Reply:
Just an idea: since its the third time we are changing this, why not to keep both?

Reply:
Data can be stored both in reservation tables and job arguments (JSON), duplicity is not an issue. However, important thing to realize is that if we do anything in the service level, it means we can error out early so the reservation request won't even start.

I am not sure in this case tho what is the benefit of resolving AMI before creating a job. Everything can fail (pubkey upload, aws launch) so in this light AMI is not any different. Why you think it is worth doing before @ezr-ondrej ?

Reply:
well just because it is easier.
We were barely able to pass the accountId to jobs, now we would need to reconstruct the Auth header so we could pass it to IB (I mean we should, I just did not want to go there)

Reply:
I don't think it is easier. 
I think we should be consistent:  fetching the ami should be as fetching the arn, this is why I chose this approach.

Reply:
oh! I get it now, we are fething it in the service, but database holds the original data, makes sense, sorry! :)
And it also will ease the transition when we decide to fetch AMI and ARN in the job itself